### PR TITLE
Publish deps

### PR DESCRIPTION
**Pull request by rofe**

### CODE REVIEW

File: scripts/home.js [hunk 1]
```diff
@@ -125,6 +125,16 @@ async function setupHomepage() {
     }
   });
 
+  // add publish dependencies
+  window.hlx=window.hlx||{};
```
Comment (kptdobe) [inline scripts/home.js:129]:
```suggestion
  window.hlx = window.hlx || {};
```

File: scripts/home.js [hunk 2]
```diff
@@ -125,6 +125,16 @@ async function setupHomepage() {
...
+  window.hlx.dependencies = [];
+  const limit = 256;
+  let offset = 0;
+  while (offset < limit * 10) {
```
Comment (kptdobe) [inline scripts/home.js:133]: This will only do the 10 first, no ?
Maybe the 10 should be a global variable.

Comment (rofe) [reply scripts/home.js:133]: Yes, I didn't want to purge too many... 

Where would you put it as a global variable?

Comment (kptdobe) [reply scripts/home.js:133]: somewhere in the file, just to make the "10" or more meaningful...

Comment (kptdobe) [reply scripts/home.js:133]: If you do not purge all, some articles disappear from the list.

Comment (rofe) [reply scripts/home.js:133]: As long as you add to the list, the only risk woyuld be duplicate cards in the first unpurged segment, but I think we have a check for that before we add a new card.

**Review by kptdobe — APPROVED** [review]

**Comment by rofe** [comment]
Now it's actually working... @kptdobe can you give it a final look please?

**Review by kptdobe — APPROVED** [review]

**Pull request merged**